src/entities/nft.ts

```diff
@@ -187,6 +188,7 @@ export function transferBase(
       timestamp,
       transactionHash
     );
+    upsertContractVolumeData(nftContract.id, value, timestamp);
```

Comment:
so this `value` here actually refers to the # of tokens being transferred, not the amount of AVAX used since we are only listening for `Transfer` events in this subgraph